imports/plugins/included/payments-stripe/register.js

```diff
@@ -11,11 +11,11 @@ Reaction.registerPackage({
     "api_key": "",
     "reaction-stripe": {
       enabled: false,
-      support: [
-        "Authorize",
-        "Capture",
-        "Refund"
-      ]
+      support: {
+        authorize: true,
```

Comment:
Why isn't de-authorize included here?

Reply:
I was following the pattern of the initial `Array` since this is an object now it is a good to make de-authorize `false`